/**:
  ImuFactory:
    ros__parameters:
      model: ACEINNA_IMU # ACEINNA_IMU or WIT_IMU

  IMU:
    ros__parameters:
      autorun: true
      imu_topic: imu
      ACEINNA_IMU:
        frame_id: imu_link
        receive_topic: CanComms/CAN2IMU
        timeout: 500000000
      WIT_IMU:
        frame_id: imu_link
        serial_port: /dev/ttyUSB0
        baudrate: 115200